keras_retinanet/models/vgg.py

```diff
@@ -49,7 +49,7 @@ def validate_backbone(backbone):
         raise ValueError('Backbone (\'{}\') not in allowed backbones ({}).'.format(backbone, allowed_backbones))
 
 
-def vgg_retinanet(num_classes, backbone='vgg16', inputs=None, **kwargs):
+def vgg_retinanet(num_classes, backbone='vgg16', inputs=None, modifier=None, **kwargs):
```

Comment:
I am not sure what `modifier` does, I only added it here to prevent `retinanet` from throwing an error because of it going into `**kwargs`.